src/main/java/duke/data/DukeDatabase.txt

```diff
@@ -0,0 +1,3 @@
+T |   | hello my name is ben
+E | X | hello you are so cool | 2012-03-03
+T |   | Ben is my name
```

Comment:
Hi, I think that the database file should not be included in the Git repository. However, if you want to make this file as a default initial database, it can also work fine!